ansible/tasks/ubuntu_desktop.yml

```diff
@@ -9,29 +9,34 @@
   ansible.builtin.apt:
     name: python3-software-properties
     state: absent
-  when: ubuntu_desktop.rc == 1
+  when: (ubuntu_desktop.rc == 1) and (not wsl_is_wsl)
```

Comment:
Do not change all these included task files but apply the `when` conditional to the place where those files are included.

Reply:
done